src/lib/types/Cart.ts

```diff
@@ -26,6 +26,7 @@ export interface Cart extends Timestamps {
 		reservedUntil?: Date;
 		depositPercentage?: number;
 		internalNote?: { value: string; updatedAt: Date; updatedById?: User['_id'] };
+		customProductName?: string;
```

Comment:
Rather than storing just the name, the chosen variations should be stored (only their ids)

eg `chosenVariations?: {name: string, value: string}[]` or `chosenVariations: Record<string, string>`